src/ui/Views/Course/Index.cshtml

```diff
@@ -101,7 +101,12 @@
               }
           </td>
           <td class="govuk-table__cell">@(course.IsRunning() ? course.GetApplicationStatus() : "")</td>
-          <td class="govuk-table__cell">@(course.IsRunning() ? course.GetHasVacancies() : "")</td>
+          <td class="govuk-table__cell">
+            @if(course.IsRunning()) {
```

Comment:
@if(course.IsRunning() && course.GetHasVacancies())

Reply:
when will `course.GetHasVacancies` be null?

Reply:
![image](https://user-images.githubusercontent.com/23801/55094099-9fad2680-50ad-11e9-96f1-36ff1172d0e6.png)

Reply:
Nevermind me I thought it was a bool